File: Golang/lista_de_exercicios_01/classes/numeros/numeros.go
package numeros

type Calculadora struct {
	Num int
}

type Matematica struct {
}

func (calc Calculadora) SomarDois(a int, b int) int {
	return a + b
}

func (calc Calculadora) SomarTres(a int, b int, c int) int {
	return a + b + c
}

func (Matematica) CalcularFatorial(x int) int {
	fatorial := 1
	for i := 1; i < x+1; i++ {
		fatorial *= i
	}

	return fatorial
}

File: Python/lista_de_exercicios_01/classes/numeros.py
class Calculadora:
    def somar_dois(self, num1: int, num2: int):
        return num1 + num2

    def somar_tres(self, num1: int, num2: int, num3: int):
        return num1 + num2 + num3


class Matematica:
    @staticmethod
    def calcular_fatorial(x):
        fatorial = 1
        for i in range(1, x+1):
            fatorial *= i

        return fatorial
